### PR TITLE
added mulled wine

### PR DESCRIPTION
**Pull request by sparsell**
Hi - thanks for the great Git PR practice.  I've added Mulled Wine to the mix.

### CODE REVIEW

File: src/recipes/mulled-wine.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        }
+    
+    ],
+    "directions": [
+        "<Combine all ingredients in large pot over medium heat.>",
+        "<Stir until all sugar (or sweetner of your choice) is dissolved.>",
+        "<Bring to a low boil and reduce heat immediately to simmer for 20 minutes.>"
+    ],
+    "image": "<drink-name.jpg>",
+    "source": "<Photo by Hannah Pemberton on Unsplash>",
```
Comment (alfg) [inline src/recipes/mulled-wine.json]: Should be the URL to the source, if you have one. If not, leave blank.

File: src/recipes/mulled-wine.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "name": "<Mulled Wine>",
```
Comment (alfg) [inline src/recipes/mulled-wine.json]:
```suggestion
    "name": "Mulled Wine",
```
You can remove all the less-than/greater-than symbols out. They are just placeholders.

Thanks!

Comment (sparsell) [reply src/recipes/mulled-wine.json]: All edits have been made - thanks very much.

Comment (sparsell) [reply src/recipes/mulled-wine.json]: Please let me know if I need to do anything further on my end.  Thanks!!

**Comment by alfg** [comment]
Thanks!

**Pull request merged**